bycycle/plts/features.py

```diff
@@ -0,0 +1,73 @@
+"""Plot cycle features."""
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+###################################################################################################
+###################################################################################################
+
+
+def plot_feature_hist(df, burst_param, ax=None, bins='auto', **kwargs):
+    """ Plot a histogram for a cycle feature.
+
+    Parameters
+    ----------
+    df : pandas.DataFrame or 1d array
```

Comment:
If this can be mixed type, then the name `df` seems odd. What about just calling it `data` or something?